### ISSUE TITLE
Train Test Validation

### ISSUE THREAD

**Issue by kleffy** [nishchaljs/3D-Hyper-UNET#2]
Hi Nishchal,

I have been reviewing your code recently. I am trying to replicate the segmentation task using a different network.

I did not see where you called your `splitTrainTestSet` or where you did some validation with the validation set.

Can please provide that information?

Thank you.

Best regards,
Daniel

**Comment by nishchaljs** [comment]
Hi Kleffy,
Thanks for showing interest in the work. I have not uploaded the test.ipynb file. But we did test it on multiple hyperspectral image datasets. The link to the paper can be found here: https://ieeexplore.ieee.org/document/9544725.
Feel free to use the in-built _train_test_split_ parameter in _model.train_ function in tensorflow for your work.

Thanks,
Nishchal

**Comment by kleffy** [comment]
Hi Nishchal,

Thank you for your response.

Best Regards,
Daniel
